app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
     <!-- Strings related to apps -->
     <string name="org_name">AeroGear</string>
     <string name="org_url">www.aerogear.org</string>
-    <string name="app_name">Android Showcase App</string>
+    <string name="app_name">Android Android Showcase</string>
```

Comment:
While android android has a nice ring to it :) it needs to be Aerogear Android Showcase.

Reply:
Nice catch!